src/components/App.js

```diff
@@ -4,15 +4,16 @@ import '../css/App.css'
 import Nav from './Nav'
 import PosterGrid from './PosterGrid'
 import MovieDetails from './MovieDetails'
-import movieData from '../movieData'
+import endpoints from '../endpoints'
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      moviePosters: movieData.movies,
+      moviePosters: [],
       showAllMovies: true,
-      moviePageId: ''
+      moviePageId: '',
```

Comment:
I really like how you named this!  It makes a lot of sense to hold this in state though (I think) Router will actually replace it.  Router v6, I'm both scared and intrigued by you.